panaroo/1.5.0/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM mambaorg/micromamba:1.5.7 as app
+
+# Version arguments
+# ARG variables only persist during build time
+# THIS ARG ONLY USED FOR A LABEL. VERSION SPECIFIED IN PANAROO-ENVIRONMENT.YML
+ARG PANAROO_SOFTWARE_VERSION="1.5.0"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:1.5.7"
+LABEL dockerfile.version="1"
+LABEL software="Panaroo"
+LABEL software.version=${PANAROO_SOFTWARE_VERSION}
+LABEL description="An updated pipeline for pangenome investigation"
+LABEL website="https://github.com/gtonkinhill/panaroo"
+LABEL license="https://github.com/gtonkinhill/panaroo/blob/master/LICENSE"
+LABEL maintainer="Kevin Libuit"
+LABEL maintainer.email="kevin.libuit@theiagen.com"
```

Comment:
Want to change lines 20 & 21 to yourself? or add `maintainer2` LABELs below?

Reply:
Oh yeah! I added myself as maintainer2